constraint_utils/constraint_basics.py

```diff
@@ -378,6 +378,29 @@ def keep_criteria(ht):
 
 def build_models(coverage_ht: hl.Table, trimers: bool = False, weighted: bool = False, half_cutoff = False,
                  ) -> Tuple[Tuple[float, float], Dict[str, Tuple[float, float]]]:
+    """
+    Build coverage model and plateau models.
+    
+    As input, `coverage_ht` is the output of `get_proportion_observed_by_coverage`
+    
+    .. note::
+        The following fields should be present in `coverage_ht`:
```

Comment:
It would be helpful to give a brief description of what these are, but not needed if the plan is to have them described well in the docstring for `get_proportion_observed_by_coverage`